pages/common/tsc.md

```diff
@@ -0,0 +1,28 @@
+# tsc
+
+> TypeScript compiler.
+> More information: <https://www.typescriptlang.org/docs/handbook/compiler-options.html>.
+
+- Compile a TypeScript file `hello.ts` into a JavaScript file `hello.js`:
```

Comment:
Perhaps change these to foo.ts and foo.js too, to be consistent with the examples ?

Reply:
Ah, my bad. Will update.